src/libs/OptionsListUtils.js

```diff
@@ -105,6 +123,7 @@ function createOption(personalDetailList, report, draftComments, {showChatPrevie
         icons: report ? report.icons : [personalDetail.avatar],
         tooltipText,
         participantsList: personalDetailList,
+        participantsSet,
```

Comment:
I don't love the idea of using a `Set` here because options get passed to a React component and it's not a great practice to pass complex things like `Set` as props. The name also doesn't really reflect what this is.. it looks like an array with all the various options for a participant's "names"?

Reply:
Cool with renaming it; in regards to complex props, are you proposing we do something different? I'm creating the Set here so we can cut down on some runtime complexity. We create the Set once here, and any time we wanna check if it has something in it it's an O(1) lookup (vs a linear scan for an array).

Reply:
Cool with renaming it; in regards to complex props, are you proposing we do something different? The usage of Set here is really just to cut down on runtime complexity. 

Also just for future reference, what type of effect can passing classes like Set as props have on React components?

Reply:
>in regards to complex props, are you proposing we do something different

Yes, use a plain object or array value.

Classes are just objects with modified prototypes, but if you say wanted to serialize the props or state at some point you'd no longer be able to do that. It's easier to document and understand the effects of passing plain objects to react components.

Reply:
I could just use a plane object, set the keys to be the names, and set their values to be true. This would still allow constant lookup without having to pass around a set.